test_data/series7_device_config.yaml

```diff
@@ -1,3 +1,10 @@
+# Which BEL names are global buffers for nextpnr?
+global_buffers:
+- BUFG
+- BUFGCTRL
```

Comment:
Aren't these the same bel, where BUFG is a subset of BUFGCTRL?

Reply:
The device resources from RapidWright has two sites, BUFG and BUFGCTRL, with BELs with the same respective name.  So while from a bitstream/hardware perspective they are the same, they are separate from the perspective of the FPGA interchange.

Reply:
All right, thanks for the clarification.